src/components/Login/Login.scss

```
@@ -0,0 +1,15 @@
+.member-service-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 373px;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  text-align: left;
+  padding: 0 34px;
+  background-color: #fff;
+  font-weight: 300;
+  //아래는 나중에 지우기
```

추후에 수정이나 추가가 필요한 경우 지금처럼 주석을 작성할 수 있습니다. 다만, 나중에 잊을 수 있으니 쉽게 검색될 수 있도록 아래와 같이 작성하시는 것이 좋습니다.
```suggestion
  //TODO: 아래는 나중에 지우기
```